hack/ci/testdata/kubermatic_backup.yaml

```diff
@@ -33,6 +33,8 @@ spec:
     replicas: 1
   ui:
     replicas: 0
+  featureGates:
+    EtcdLauncher: true
```

Comment:
This is going to be a problem because `pre-kubermatic-etcd-launcher-e2e` is specifically built to test the "plain" `etcd` to `etcd-launcher` migration. Applying this change, all clusters created in the e2e test will be created with `etcd-launcher` by default and we won't notice if a PR breaks the migration path.

Perhaps backups need to have their separate e2e tests.

Reply:
Or maybe the e2e test needs two `Seeds`.

Reply:
I see. Let me check the test. Maybe  it can be made immune to such changes.